content/docs/tooling/cli/_index.md

```diff
@@ -59,6 +61,481 @@ Outputs help text
+
+{{< collapse id="dellctl-install-cli-params" title="From-File Parameters" card="false" >}}
+{{<table install-flags >}}
+| Parameter               | Description | Default | Mandatory |
+|:-------------------|:------------|:--------|:-----------
+| certified                                  | The certified images from registry.connect.redhat.com are used instead of the quay.io images. | false | No |
+| config-version <string>                    | Version of Container Storage Modules to install. | {{VERSION}} | No |
+| csi-node-prefix <string>                   | The prefix for all CSI nodes provisioned by the driver. | csi-node | No |
+| csi-volume-prefix <string>                 | The prefix for all CSI volumes provisioned by the driver. | csivol | No |
+| force                                      | The existing Container Storage Module resources are deleted and then recreated.  | false | No |
+| from-file <string>                         | Path to a YAML file containing configuration details for installing the CSM.  | "" | No |
+| machineconfig                              | Configure pre-requisities based on the provided block-protocol parameter. See the relevant orchestrator and platform [installation page](../../getting-started/installation/_index.md) for which prerequisites are configured.  | false | No |
+| modules <stringArray>                      | Container Storage Modules modules to install. Supported modules are: replication, authorization, observability, resiliency.  | "" | No |
+| namespace <string>                         | Namespace to install into (lowercase alphanumeric, may include dashes, must start/end with alphanumeric).  | "namespace" | Yes |
+| operator-install                           | Container Storage Modules Custom Resource Definitions will be installed.  | false | No |
+| output                                     | Output from dellctl install. This dumps every generated yaml to the console. | false | No |
+| registry-url <string>                      | Registry URL to use for images.  | "" | No |
+| snapshot-controller                        | Configure snapshot CRDs and controller.  | false | No |
+| validate-connectivity                      | Run a DaemonSet on all nodes to verify connectivity to storage systems.  | false | No |
+| **storage**                                | This section configures the storage systems. It is provided as an array. | - | - |
+| storage.endpoint                           | The IP address or hostname of the storage system endponit (i.e., 10.0.0.1). | "" | Yes |
+| storage.username                           | Username for accessing PowerFlex system. If authorization is enabled, username will be ignored. | "" | Yes |
+| storage.block-protocol                     | Transport protocol for block storage (Fc, ISCSI, NVMeTCP, NVMeFC, None, auto) | FC | No |
+| storage.nfs-acls                           | NFS ACLs used if NFS is being used on the array. | 0777 | No |
```

Comment:
include-nas-servers and exclude-nas-servers parameters for storage are missing. We can add them near nfs-acls